lib/module.js

```diff
@@ -3,6 +3,11 @@ const { resolve } = require('path')
 module.exports = function module (moduleOptions) {
   const options = this.options['google-analytics'] || moduleOptions
 
+  // Don't include when no analytics id is given
+  if (!(options.id || options.ua)) {
```

Comment:
I will also use only `options.id` and after the check of `options.ua` below

Reply:
good catch, ye, it totally make sense :)